internal/commands/app/create_inputs_test.go

```diff
@@ -709,6 +709,92 @@ func TestAppCreateInputsResolveCluster(t *testing.T) {
 		assert.Equal(t, errors.New("client error"), err)
 		assert.Equal(t, "123", expectedGroupID)
 	})
+
+	t.Run("should error when more than one cluster passed in when creating template app", func(t *testing.T) {
+		_, ui := mock.NewUI()
+		clusterNames := []string{"Cluster0", "Cluster1"}
+
+		ac := mock.AtlasClient{}
+		ac.ClustersFn = func(groupID string) ([]atlas.Cluster, error) {
+			return []atlas.Cluster{
+				{ID: "456", Name: clusterNames[0]},
+			}, nil
+		}
+		inputs := createInputs{
+			newAppInputs: newAppInputs{
+				Template: "ios.template.todo",
+			},
+			Clusters: clusterNames,
+		}
+		_, _, err := inputs.resolveClusters(ui, ac, "123")
+		assert.Equal(t, "template apps can only be created with one data source", err.Error())
```

Comment:
[opt/nit] for here and the other spots if you go the route
```suggestion
		assert.Equal(t, errors.New("template apps can only be created with one data source"), err)
```